backend/database_handler/transactions_processor.py

```
@@ -1,20 +1,29 @@
 from sqlalchemy.orm.attributes import flag_modified
+
+from backend.node.types import Vote, Receipt
+from .models import Transactions, TransactionStatus
 from eth_utils import to_bytes, keccak, is_address
+import json
+import base64
+import time
+from backend.domain.types import TransactionType
 from web3 import Web3
+import os
+from enum import Enum
```

Ruff (0.11.9)

Redefinition of unused `Enum` from line 3

Remove definition: `Enum`

(F811)